src/apis/acquisition_api.py

```diff
@@ -1,65 +1,132 @@
-"""Module"""
+"""Module to send requests"""
 
 # Libraries
 import os
-from dataclasses import dataclass
 import requests
-from io import BytesIO
-from collections import deque
-import cv2
-
-@dataclass
-class AcquisitionsAPI:
-    """Class"""
-
-    def receive_data(self):
-        """b"""
-
-        cap = cv2.VideoCapture(0)
-        _, frame = cap.read()
-
-        file = BytesIO()
-        timestamp = "2022-02-15T15:00:00.000000"
-        file_name = f"{timestamp}.jpg"
-        path = f"/tmp/{file_name}"
-
-        cv2.imwrite(path, frame)
-        multipart_form_data = {
-            "store_id": (None, 4),
-            "timestamp": (None, timestamp),
-            "process": (None, "gender"),
-            "status": (None, "running"), # Optional
-            "tracking_id": (None, 15), # Optional
-            "centroid": (None, "50,50"), # Optional
-            "capture": (file_name, open(path, "rb")),
-        }
-
-        return multipart_form_data
-    
-    def send_api_data(self, multipart_form_data):
-        """ """
-
-        # url = os.environ.get("ACQUISITION_API", None)
-        url = "https://dev.vico.ai/acquisition/monito"
-        response = requests.post(url=url, files=multipart_form_data)
+import redis
+from rq import Queue
+import logging
+import pickle
+from dotenv import load_dotenv, find_dotenv
+
+
+class ConsumeAPI():
```

Comment:
Aquí sugeriría cambiar el nombre de la clase.

Como lo indicas en el comentario: """Class to consume an API and queue requests"""

Esta clase no sólo consume el api, lo importante es que encola los requests.

Yo le pronfría algo así como RequestQueuer()